examples/demos/elation/Availability.jsx

```diff
@@ -0,0 +1,36 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+export default class Availability extends React.Component {
+  static propTypes = {
+    availability: PropTypes.object,
+  }
+
+  render() {
+    const { availability} = this.props;
+    const nowStartTime = new Date(Date.now());
```

Comment:
Isn't `new Date(Date.now())` the same thing as `new Date()`?

Reply:
🤦